deploy/example/resizing/storageclass-azuredisk-csi-resize.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: disk.csi.azure.com
-provisioner: disk.csi.azure.com
-allowVolumeExpansion: true
```

Comment:
If `allowVolumeExpansion ` does not set to true, volume expansion feature will not work.

Reply:
I have set this parameter as true in `deploy/example/storageclass-azuredisk-csi.yaml`, don't need to create a new yaml for this.